src/libraries/System.Text.Json/tests/Common/RequiredKeywordTests.cs

```diff
@@ -0,0 +1,597 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
```

Comment:
This is @krwq's original test suite for require member support, moved into and adapted for the `Common` tests folder. Unfortunately github's diff shows this as a new file 😕

Reply:
If you're concerned about git history I believe it is still preserved, so that should be fine.

Reply:
Makes sense, although it doesn't make reviewing easier :-/